### PR TITLE
Jira service management connector

### PR DESCRIPTION
**Pull request by jaykayudo**
/claim #2281

## Description
[Provide a brief description of the changes in this PR]

## How Has This Been Tested?
[Describe the tests you ran to verify your changes]

## Accepted Risk
[Any know risks or failure modes to point out to reviewers]

## Related Issue(s)
[If applicable, link to the issue(s) this PR addresses]

## Checklist:
- [ ] All of the automated tests pass
- [ ] All PR comments are addressed and marked resolved
- [ ] If there are migrations, they have been rebased to latest main
- [ ] If there are new dependencies, they are added to the requirements
- [ ] If there are new environment variables, they are added to all of the deployment methods
- [ ] If there are new APIs that don't require auth, they are added to PUBLIC_ENDPOINT_SPECS
- [ ] Docker images build and basic functionalities work
- [ ] Author has done a final read through of the PR right before merge

### CODE REVIEW

**Comment by jaykayudo** [comment]
@Weves check this out

File: web/src/lib/sources.ts [hunk 1]
```diff
@@ -111,6 +111,12 @@ const SOURCE_METADATA_MAP: SourceMap = {
     category: SourceCategory.ProjectManagement,
     docs: "https://docs.danswer.dev/connectors/jira",
   },
+  jira_service_management: {
```
Comment (Weves) [inline web/src/lib/sources.ts:114]: I believe Jira Service Management has a different icon (see the icon [here](https://www.atlassian.com/software/jira/service-management?&aceid=&adposition=&adgroup=140639872753&campaign=17732529748&creative=639445349742&device=c&keyword=jira%20service%20management&matchtype=e&network=g&placement=&ds_kids=p72254545591&ds_e=GOOGLE&ds_eid=700000001721198&ds_e1=GOOGLE&gad_source=1&gclid=Cj0KCQjw_sq2BhCUARIsAIVqmQsY-qw1JjOv6Cx9A-pTQxfvXjlVprEkFySFVZc32heJFob7IdCPXQ0aAlJjEALw_wcB&gclsrc=aw.ds). Can we use that one instead?

**Review by Weves — CHANGES_REQUESTED** [review]
@jaykayudo overall looks reasonable! Can we get:

1. A video of the new connector UI / using the UI to create an instance of this connector 
2. A connector test under the `daily` directory.

More details on both of these here: https://github.com/danswer-ai/danswer/blob/main/backend/danswer/connectors/README.md#before-opening-pr

File: web/src/lib/sources.ts (hunk 1 above)
Comment (jaykayudo) [reply web/src/lib/sources.ts:114]: okay... i will get right on it

Comment (Weves) [reply web/src/lib/sources.ts:114]: @jaykayudo any updates here?

Comment (jaykayudo) [reply web/src/lib/sources.ts:114]: @Weves I have changed the icon and i have added a unit test. you can see that by reviewing the file changes. But about the video, i have been having issues installing some of the dependencies for danswer so i have not been able to do it. Please is there a docker compose for everything that makes it easier to run?

Comment (Weves) [reply web/src/lib/sources.ts:114]: @jaykayudo yes! Have you checked out our quickstart guide: https://docs.danswer.dev/quickstart. 

This should get things up and running locally!

**Pull request closed without merging**